installers/mac/tar/templates/Info.plist.template

```diff
@@ -17,11 +17,11 @@
         <key>CFBundlePackageType</key>
         <string>BNDL</string>
         <key>CFBundleShortVersionString</key>
-        <string>1.0</string>
+        <string>@major@.@minor@.@security@</string>
         <key>CFBundleSignature</key>
         <string>????</string>
         <key>CFBundleVersion</key>
-        <string>11.@minor@.@security@</string>
```

Comment:
This line has been changed in #69 and merged.

Reply:
I believe because the resulting merge is identical, there won't be any conflicts.